packages/backend/src/clients/dbtCloud/DbtCloudGraphqlClient.ts

```diff
@@ -83,26 +92,25 @@ export default class DbtCloudGraphqlClient {
     }
 
     // eslint-disable-next-line class-methods-use-this
-    async runGraphQlQuery<T>({
-        domain,
-        query,
-        environmentId,
-        bearerToken,
-    }: RunGraphQLQueryFnArgs): Promise<T> {
-        return DbtCloudGraphqlClient.getClient({ domain, bearerToken }).request(
-            query,
+    async getViews(): Promise<SemanticLayerView[]> {
+        return this.transformers.viewsToSemanticLayerViews([
             {
-                environmentId,
+                label: 'DBT Semantic View',
+                name: 'dbtSemanticView',
```

Comment:
is this the official name?

Reply:
DBT doesn't have this concept so here we just return a placeholder object in case we want to display it in the FE